File: portfolio-server/src/main/java/com/maksudsharif/portfolioserver/model/Education.java
package com.maksudsharif.portfolioserver.model;

import java.io.Serializable;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class Education implements Serializable
{

  private String title;
  private String addendum;
  private String institution;
  private String range;
}

File: portfolio-client/src/app/components/model/education.ts
import {EducationItem} from './educationItem';

export class Education {
  public educations: EducationItem[];

  constructor(education: any[]) {
    this.educations = [];
    for (const ed of education) {
      this.educations.push(EducationItem.of(ed.title, ed.addendum, ed.institution, ed.range));
    }
  }
}
